wx-base/src/lib.rs

```diff
@@ -306,6 +306,20 @@ impl<const OWNED: bool> Drop for ArrayStringIsOwned<OWNED> {
     }
 }
 
+impl<T> From<T> for ArrayString
+where
+    T: IntoIterator,
+    T::Item: std::fmt::Display,
+{
+    fn from(items: T) -> Self {
+        let array_string = Self::new();
+        for item in items {
+            array_string.add(&format!("{item}"));
```

Comment:
Converting from Display trait is reasonable.

I prefer use auto impl'd `ToString.to_string()` as more intuitive and may be more performant because it doesn't format actually.